source/portable/NetworkInterface/xilinx_ultrascale/NetworkInterface.c

```diff
@@ -129,6 +129,25 @@ static BaseType_t prvGMACWaitLS( TickType_t xMaxTimeTicks );
  */
 static void prvEMACHandlerTask( void * pvParameters );
 
+/* FreeRTOS+TCP/multi :
+ * Each network device has 3 access functions:
+ * - Initialise the device
+ * - Output a network packet
+ * - Return the PHY Link-Status (LS)
+ * They can be defined as static because the function addresses
+ * addresses will be stored in struct NetworkInterface_t. */
+
+static BaseType_t xNetworkInterfaceInitialise( NetworkInterface_t * pxInterface );
```

Comment:
These 3 functions normally have a name that starts like `xCyclone_NetworkInterfaceInitialise()`. Even though these are static functions, I would like to give them unique names.
For instance `xUltraScale_NetworkInterfaceInitialise()`.

( the reason: sometimes I use grep to find a particular NIC function. Then I don't like to see 56 hits that all look similar )